CMakeLists.txt

```diff
@@ -89,8 +89,9 @@ add_library(pyyjson SHARED
     src/simd/cvt.c
     src/simd/mask_table.c
     src/tls.c
+    third_party/ryu/d2s.c
 )
-target_include_directories(pyyjson PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src> ${Python3_INCLUDE_DIRS})
+target_include_directories(pyyjson PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src> ${CMAKE_CURRENT_SOURCE_DIR}/third_party ${Python3_INCLUDE_DIRS})
```

Comment:
Remove the third_party dir, put all the files under `src`, just like `xxhash.h`. Don't modify include directory